matrix-sdk/src/main/java/org/matrix/androidsdk/data/EventTimeline.java

```diff
@@ -446,6 +446,10 @@ public void handleJoinedRoomSync(RoomSync roomSync, boolean isGlobalInitialSync)
             state.setDataHandler(mDataHandler);
 
             this.mBackState = this.mState = state;
+
+            // in the pending invitation case the room sends only very little informations
+            // that is why we can consider that it is ready
+            mRoom.setReadyState(true);
```

Comment:
This is not the right place to add this change.
Here the room was just joined. This is not an invite anymore

Your change should be added in `handleInvitedRoomSync`,
after the events handling